CMakeLists.txt

```diff
@@ -490,7 +490,7 @@ elseif(CELERITAS_USE_HIP)
   find_library(ROCTX_LIBRARY roctx64)
   if(ROCTX_LIBRARY)
     set(CELERITAS_HAVE_ROCTX ON)
-    target_link_libraries(celeritas_device_toolkit INTERFACE ${ROCTX_LIBRARY})
+    target_link_libraries(celeritas_device_toolkit INTERFACE "${ROCTX_LIBRARY}")
```

Comment:
Why are the quote needed here?

Reply:
In the unlikely case that there's a space in the path: it's a good practice.

Reply:
Fair enough.  There are other place where we forgot to do this :)